fix: Refine the color of the show password icon button

floraSimpleDev opened the pull request:
fixed the user login/register page:
- add hover and click effects for the show password icon button (eye icon button)
Resolve CP-84

floraSimpleDev committed:
fix: Refine the color of the show password icon button
fixed the user login/register page:
- add hover and click effects for the show password icon button (eye icon button)
Resolve CP-84

Chris-Wang reviewed (changes requested):
I like the idea to refine the colour on the eye button, now it's more dynamic and user-friendly.

To make this button look better, I believe, it should show a higher contrast colors to differentiate the active/inactive button status.

Also, please attach screenshot of your update in description;
and need to 
- update `Story point estimate` and `Due date` of ticket on Jira
- remember to drag your ticket from backlog to current sprint
<img width="1052" alt="image" src="https://github.com/user-attachments/assets/bf03f7d8-993a-44b6-9d7d-ba7a11431785">

floraSimpleDev committed:
fix: Refine the color of the show password icon button
fixed the user login/register page:
- add hover and click effects for the show password icon button (eye icon button)
Resolve CP-84

floraSimpleDev committed:
fix: Refine the color of the show password icon button
fixed the user login/register page:
- add hover and click effects for the show password icon button (eye icon button)
Resolve CP-84

floraSimpleDev committed:
Merge branch 'fix/CP-84-Refine-the-color-of-the-show-password-icon-button' of https://github.com/BeeQuantAI/platform_app into fix/CP-84-Refine-the-color-of-the-show-password-icon-button

Chris-Wang reviewed (changes requested):
Need squash your commits into one
